src/playground/addons/story-utils/index.js

```diff
@@ -539,95 +539,113 @@ export const correctSvgPath = (data) => {
 };
 
 export const getFormControls = (data, type) => {
-  return {
-    label: {
-      type: { name: 'string', required: true },
-      defaultValue: data.label,
-      description: `Label of the form ${type}`,
-      table: {
-        type: { summary: 'string' },
-        defaultValue: { summary: '' },
-        category: 'Content',
-      },
+  const argTypes = {};
```

Comment:
You refactored the whole function to add the condition for the width.. It's ok, eh, but you could have created the argTypes object directly instead of initialising as an empty object for the common ones:
```
const argTypes = {
 label: {},
helper_text: {}
...
};
```

and then:
```
if (data.width) {
  argTypes.width = {};
}
```
Anyway, as i said, nothing you have to necessarily change this way.

Reply:
True, I like to have all the controls defined in the same way ;)